api/pyproject.toml

```diff
@@ -12,6 +12,7 @@ psycopg2 = "^2.8"
 sentry-sdk = "^0.12.3"
 django-extensions = "^2.2"
 django-waffle = "=0.17.0"
+model_mommy = "=1.6.0"
```

Comment:
This should be in the dev dependencies. :D

Reply:
Moved Laew